### ISSUE TITLE
Conflicted with quickfix-reflector plugin

### ISSUE THREAD

**Issue by markwu** [ronakg/quickr-preview.vim#17]
I installed both https://github.com/stefandtw/quickfix-reflector.vim and https://github.com/ronakg/quickr-preview.vim plugins, I found when I grep and open quickfix window, I will get the following messages:

![2019-02-07 8 23 45](https://user-images.githubusercontent.com/196861/52411381-90324800-2b16-11e9-82b4-c66c300b8ebf.png)

I have no idea what's going on. The only way to get rid of the error messages is to comment the following line:

`"    autocmd BufDelete <buffer> pclose`

And change the close qf mapping to

`nnoremap <silent> <plug>(quickr_preview_qf_close) :cclose<CR>:lclose<CR>:pclose<CR>`

if you could spare some time to investigate this issue, that will be great.

Thanks.

**Comment by matt1003** [comment]
I have also been having problems with the `BufDelete` auto-command. Do you still have the issue if you change it to `BufUnload` instead?

**Comment by markwu** [comment]
Yes, even change to `BufUnload`, I still have the issue. Anyway, I already remove it.
